### ISSUE TITLE
Elevated urinary 5-amino-4-imidazolecarboxamide-riboside level

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9617]
**Preferred term label:**
Elevated urinary 5-amino-4-imidazolecarboxamide-riboside level

**Synonyms**
Elevated urinary AICA-riboside level

**Definition (free text, please give PubMed ID)**
Increased amount of 5-amino-4-imidazolecarboxamide(AICA)-riboside in the urine. AICA-riboside is the dephosphorylated counterpart of AICAR, an intermediate of de novo purine biosynthesis.PMID:15114530

**Parent term (use hpo.jax.org/app)**
Abnormal urinary purine level HP:0033161

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
608688 
PMID:15114530  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
